containerbuddy/reload-memcached.sh

```diff
@@ -1,7 +0,0 @@
-#!/bin/bash
```

Comment:
is there a reason to be inconsistent in the naming?

- `reload-db.sh`
- `memcached.sh`